rust/exchange_rate/canister_ids.json

```diff
@@ -0,0 +1,11 @@
+{
+  "__Candid_UI": {
+    "ic": "a4gq6-oaaaa-aaaab-qaa4q-cai"
+  },
+  "exchange_rate": {
+    "ic": "zs5rc-myaaa-aaaao-qaaba-cai"
+  },
+  "exchange_rate_assets": {
+    "ic": "zv4xw-baaaa-aaaao-qaabq-cai"
+  }
+}
```

Comment:
`dfx deploy --network ic` will try to deploy to these canisters. I think it's best to delete this file

Reply:
Same as above